src/components/binding/binding-loader.js

```diff
@@ -87,14 +92,14 @@ genotet.BindingLoader.prototype.loadFullTrack = function(trackIndex, fileName,
   };
   this.get(genotet.data.serverURL, params, function(data) {
     var track = {
+      gene: data.gene,
       fileName: fileName,
       overview: data,
       detail: data
     };
-    var addTrack = trackIndex == this.data.tracks.length;
     this.data.tracks[trackIndex] = track;
     this.updateRanges_();
-    if (addTrack) {
+    if (isAddTrack) {
       // Add one more track.
       this.loadBindingList();
```

Comment:
I think loadBindingList() queries the server for the list of available binding data files to show in the panel so that you can switch genes. However I don't see why we should do that when isAddTrack. 

Looks like @djasmine added this [here](https://github.com/ViDA-NYU/genotet/blob/d1a7e9b554ca7355b731116099edf9b673107f1d/components/binding/binding-loader.js#L100) and loadBindingList actually fired 'track' [here](https://github.com/ViDA-NYU/genotet/blob/d1a7e9b554ca7355b731116099edf9b673107f1d/components/binding/binding-loader.js#L251). However I don't think signal('track') makes sense in loadBindingList as they are quite independent. Getting binding list doesn't seem to be a proper reaction of user modifying a track.